isis/src/base/apps/csminit/csminit.cpp

```diff
@@ -465,8 +465,11 @@ namespace Isis {
         cube->write(originalCameraStatistics);
       }
 
-      if (originalFootprint.Polys()->getNumGeometries() != 0) {
-        cube->write(*(originalFootprint.toBlob()));
+
+      if (originalFootprint.Polys() != NULL){
+        if (originalFootprint.Polys()->getNumGeometries() != 0) {
```

Comment:
This should just be `if (originalFootprint.Polys() != NULL && originalFootprint.Polys()->getNumGeometries() != 0)